README.md

```diff
@@ -1,12 +1,135 @@
-# deqm-test-server
+# Data Exchange for Quality Measures (DEQM) Test Server
+
+Test server for executing FHIR-based Electronic Clinical Quality Measures (eCQMs).
+
+- [Installation](#installation)
+
+  - [Prerequisites](#prerequisites)
+  - [Local Installation](#local-installation)
+  - [Testing](#testing)
+  - [MongoDB](#mongodb)
+
+- [Usage](#usage)
+
+  - [Database Setup](#database-setup)
+  - [CRUD Operations](#crud-operations)
+  - [Searches] (#searches)
+  - [Supported FHIR Operations](#supported-fhir-operations)
+
+- [License](#license)
+
+## Installation
+
+### Prerequisites
+
+- [Node.js >=10.15.1](https://nodejs.org/en/)
```

Comment:
Suggest adding a section here with different prereqs for doing the docker installation, since that won't require installing mongo, node, etc.

It can just say how to run the server with docker (`docker-compose up`)